src/resources/views/corporation/tracking.blade.php

```diff
@@ -37,12 +39,22 @@
                 {!! img('character', $character->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
                 <span rel="id-to-name">{{ $character->character_id }}</span>
               </a>
+            </td>
+            {{--<td>
               @if(!is_null($character->ship_type_id))
                 <i class="pull-right" data-toggle="tooltip"
                    title="" data-original-title="{{ $character->type->typeName }}">
                   {!! img('type', $character->ship_type_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
```

Comment:
Please restore this.

Reply:
I came to the conclusion that the ship type is useless without the `last known location`. Therefore i have commented this part to later when we fixxed the location tracking.

![image](https://user-images.githubusercontent.com/6583519/43364287-af5783cc-9317-11e8-92a3-1c1be2a89894.png)

I have screenshotted if i would add this and find this confusing